config/routes.rb

```diff
@@ -37,7 +37,7 @@ def matches?(_)
           get 'eligibility' => 'listings#eligibility'
         end
       end
-      get 'trk' => 'email#confirmation'
```

Comment:
what is trk?

Reply:
It is short for tracking. Email templates in salesforce campaign monitor have a size limit for links and the token itself was getting long so we decided to use shorter names in the api like `trk` for tracking and the param `t` for token.